cadc-storage-adapter-ad/src/intTest/java/org/opencadc/inventory/storage/ad/AdStorageAdapterIteratorTest.java

```diff
@@ -208,6 +204,78 @@ public Object run() throws Exception {
         });
     }
 
+    @Test
+    public void testIteratorVOSpac() throws Exception {
+        Subject.doAs(testSubject, new PrivilegedExceptionAction<Object>() {
+
+            // Should be VOSpac except that the buckets in that archive are too large
```

Comment:
this is quite complex to look at and the overridable method is ugh... now that we are really doing 4-char buckets, a single 4 char bucket has ~3700 files, which is about the same as the IRIS test in `testIteratorOrder()`... that would let you get rid of `getVOSpac()` and use a constant (or, to be honest, I don't mind using hard coded strings that are only in one class... sometimes easier to see exactly what's going on)

Reply:
Makes sense. Tested it and there are no significant performance hits in the int test.